<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QmitkDiffusionQuantificationViewControls</class>
 <widget class="QWidget" name="QmitkDiffusionQuantificationViewControls">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>343</width>
    <height>888</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>QmitkTemplate</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QGroupBox" name="m_InputData">
     <property name="title">
      <string>Please Select Input Data</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Image</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="m_InputImageLabel">
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#ff0000;&quot;&gt;mandatory&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_4">
     <property name="title">
      <string>General Parameters</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Scale Image Values:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QDoubleSpinBox" name="m_ScaleImageValuesBox">
        <property name="maximum">
         <double>9999999.000000000000000</double>
        </property>
        <property name="value">
         <double>1.000000000000000</double>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>Raw diffusion-weighted image</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QCommandLinkButton" name="m_AdcDwiButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>ADC</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCommandLinkButton" name="m_MdDwiButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>MD</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCommandLinkButton" name="m_BallStickButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Ball-Stick</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCommandLinkButton" name="m_MultiTensorButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Multi-Tensor Fit</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Q-Ball image</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QFrame" name="frame">
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QCheckBox" name="m_StandardGFACheckbox">
           <property name="text">
            <string>Generalized GFA</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QFrame" name="frame_2">
           <property name="frameShape">
            <enum>QFrame::NoFrame</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QLabel" name="m_ParamK">
              <property name="enabled">
               <bool>true</bool>
              </property>
              <property name="text">
               <string>k</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="m_ParamKEdit">
              <property name="enabled">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="m_ParamP">
              <property name="enabled">
               <bool>true</bool>
              </property>
              <property name="text">
               <string>p</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="m_ParamPEdit">
              <property name="enabled">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QCommandLinkButton" name="m_GFAButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>GFA</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="frame_3">
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Min. angle</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="MinAngle"/>
         </item>
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Max. angle</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="MaxAngle"/>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QCommandLinkButton" name="m_CurvatureButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Curvature</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Tensor image</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QCommandLinkButton" name="m_FAButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>FA (Fractional Anisotropy)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCommandLinkButton" name="m_RAButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>RA (Relative Anisotropy)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCommandLinkButton" name="m_ADButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>AD (Axial Diffusivity)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCommandLinkButton" name="m_RDButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>RD (Radial Diffusivity)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCommandLinkButton" name="m_MDButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>MD (Mean Diffusivity)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCommandLinkButton" name="m_ClusteringAnisotropy">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>1-(λ2+λ3)/(2*λ1)</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="spacer1">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Expanding</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>220</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <includes>
  <include location="local">QmitkDataStorageComboBox.h</include>
 </includes>
 <resources/>
 <connections/>
</ui>
